src/imitation/rewards/reward_nets.py

```diff
@@ -105,17 +105,16 @@ def preprocess(
 
         return state_th, action_th, next_state_th, done_th
 
-    def predict(
+    def predict_th(
         self,
         state: np.ndarray,
         action: np.ndarray,
         next_state: np.ndarray,
         done: np.ndarray,
     ) -> np.ndarray:
```

Comment:
`) -> th.Tensor:`

Reply:
Yeah good point. Im sad `pytype` didn't complain about this ;(